ask_a_librarian/models.py

```diff
@@ -16,9 +18,22 @@ class AskPage(PublicBasePage, ContactFields):
     Page type for Ask A Librarian pages.
     """
 
+    intro = StreamField(
+        [
+            ('paragraph', RichTextBlock()),
+            ('reusable_content_block', ReusableContentBlock()),
+            ('html', RawHTMLBlock()),
+        ],
+        null=True,
+        blank=True,
+    )
     ask_widget_name = models.CharField(max_length=100, blank=True)
-    reference_resources = RichTextField(blank=True)
     body = StreamField(DefaultBodyFields())
+    reference_resources = RichTextField(
+        blank=True,
+        help_text=
+        'Links to guide links and other Ask pages. Make new sections with Header 3'
```

Comment:
This isn't your fault, it's something annoying the autofixer is doing right now. We can get rid of the linting error here by doing something like:

```
    reference_resources = RichTextField(
        blank=True,
        help_text='Links to guide links and other \
        Ask pages. Make new sections with Header 3'
    ) 
```
After you format the text with the ecape `\` the autofixer will stop dropping it down to the new line.